src/main/java/com/monumental/services/MonumentService.java

```diff
@@ -75,28 +82,45 @@ private Predicate buildSimilarityQuery(CriteriaBuilder builder, CriteriaQuery qu
     /**
      * Creates a PostGIS ST_DWithin query on the Monument's point field and adds it to the specified CriteriaQuery
      * @param builder The CriteriaBuilder for the query
+     * @param query The CriteriaQuery being created
      * @param root The Root associated with the CriteriaQuery
      * @param latitude The latitude of the point to compare to
      * @param longitude The longitude of the point to compare to
-     * @param miles - The number of miles from the comparison point to check
+     * @param miles The number of miles from the comparison point to check
+     * @param orderByDistance If true, results will be ordered by distance ascending
      */
-    private Predicate buildDWithinQuery(CriteriaBuilder builder, Root root, Double latitude, Double longitude, Integer miles) {
+    private Predicate buildDWithinQuery(CriteriaBuilder builder, CriteriaQuery query, Root root, Double latitude, Double longitude,
+                                        Integer miles, Boolean orderByDistance) {
         String comparisonPointAsString = "POINT(" + longitude + " " + latitude + ")";
         Integer feet = miles * 5280;
 
+        Expression[] expressions = new Expression[]{
+            builder.function("ST_Transform", Geometry.class, root.get("coordinates"),
+                builder.literal(feetSrid)
+            ),
+            builder.function("ST_Transform", Geometry.class,
+                builder.function("ST_GeometryFromText", Geometry.class,
+                    builder.literal(comparisonPointAsString),
+                    builder.literal(coordinateSrid)
+                ),
+                builder.literal(feetSrid)
+            ),
+            builder.literal(feet)
+        };
```

Comment:
Can you add some comments to this section so it's easier to see what `expressions[0]` is in the inline array declaration?

Reply:
I changed them to variables because what even in the hell was I doing before, I don't know